its/plugin/tests/src/test/java/com/sonar/it/java/suite/JavaClasspathTest.java

```diff
@@ -78,6 +78,7 @@ public void deleteData() {
   public void should_use_new_java_binaries_property() {
     SonarScanner scanner = ditProjectSonarScanner();
     scanner.setProperty("sonar.java.binaries", "target/classes");
+    TestUtils.provisionProject(ORCHESTRATOR, PROJECT_KEY_DIT,PROJECT_KEY_DIT,"java","dit-check");
```

Comment:
Next time try to reformat the code to have the standard spacing between function arguments